hack/lib/catalogsource.bash

```diff
@@ -67,12 +67,24 @@ function install_catalogsource {
     mv "${rootdir}/_output/bkp.Dockerfile" "${index_build_dir}/Dockerfile"
   fi
 
+# IMAGE_INDEX_BUNDLE decleare in var.bash as default null then work like existing if not null it will take the value from script iib images
+  if [ -v IMAGE_INDEX_BUNDLE ] && [ ! -z "$IMAGE_INDEX_BUNDLE" ]; then
```

Comment:
You previously had `if [ ! -n "${IMAGE_INDEX_BUNDLE:-}" ]; then` but it should have been `if [ -n "${IMAGE_INDEX_BUNDLE:-}" ]; then`  (note the missing exclamation mark)
The `-n` means: If the variable is non-empty, do something.